app/src/main/java/com/github/se/gomeet/ui/mainscreens/Explore.kt

```diff
@@ -324,62 +327,65 @@ private fun ContentInColumn(
     Column {
       TopTitle(forColumn = true, alpha = columnAlpha)
 
-      LazyColumn(modifier = Modifier.alpha(columnAlpha), state = listState) {
-        itemsIndexed(events) { _, event ->
-          Column {
-            Card(
-                elevation = 4.dp,
-                modifier =
-                    Modifier.size(width = 360.dp, height = 200.dp).padding(8.dp).clickable {
-                      nav.navigateToEventInfo(
-                          eventId = event.eventID,
-                          title = event.title,
-                          date = getEventDateString(event.date),
-                          time = getEventTimeString(event.time),
-                          description = event.description,
-                          organizer = event.creator,
-                          loc = LatLng(event.location.latitude, event.location.longitude),
-                          rating = 0.0 // TODO: replace with actual rating
-                          // TODO: add image
-                          )
-                    }) {
-                  val painter: Painter =
-                      if (event.images.isNotEmpty()) {
-                        rememberAsyncImagePainter(
-                            ImageRequest.Builder(LocalContext.current)
-                                .data(data = event.images[0])
-                                .apply(
-                                    block =
-                                        fun ImageRequest.Builder.() {
-                                          crossfade(true)
-                                          placeholder(R.drawable.gomeet_logo)
-                                        })
-                                .build())
-                      } else {
-                        painterResource(id = R.drawable.gomeet_logo)
-                      }
-                  Image(
-                      painter = painter,
-                      contentDescription = "",
-                      modifier = Modifier.fillMaxSize(),
-                      alignment = Alignment.Center,
-                      contentScale = ContentScale.Crop)
+      LazyColumn(
```

Comment:
The lazy column of events looks better than before. I would suggest rounding the corners of the images to match the ones in the eventInfo screen